src/components/Drawer/index.js

```diff
@@ -122,32 +124,37 @@ export default function Drawer(props) {
                     tabIndex={-1}
                     aria-labelledby={headerId}
                     aria-modal
-                    aria-hidden
+                    aria-hidden={!drawerIsOpen}
                     aria-describedby={contentId}
                     className={className}
                     isOpen={drawerIsOpen}
                     style={style}
                     size={size}
-                    slideFrom={slideFrom}
+                    slideFrom={getSlideFrom(slideFrom, 'left')}
                     ref={drawerRef}
                     onAnimationEnd={onSlideEnd}
                 >
-                    <Header content={header} />
+                    <Header id={headerId} content={header} />
```

Comment:
we need to pass id to a dom element in order to associate the underlying element of this component with the drawer container through aria-labelledby attribute